lib/spack/spack/package.py

```diff
@@ -545,6 +545,9 @@ class SomePackage(Package):
     # Verbosity level, preserved across installs.
     _verbose = None
 
+    # Comma separated list of GitHub usernames of package maintainers
+    maintainers = ''
```

Comment:
Why have it be comma-separated and not a list of strings?

Reply:
I am happy to change to 
```
maintainers = ['name1', 'name2']
```
if that's what you prefer.

Reply:
Rather than messing about with splitting and joining, I think this just giving the split is easier.